src/ansys/openapi/common/_base/_types.py

```diff
@@ -31,13 +32,52 @@ class ModelBase(metaclass=abc.ABCMeta):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         ...
 
-    @abc.abstractmethod
-    def to_dict(self) -> Dict[str, DeserializedType]:
-        ...
+    def to_dict(self) -> Dict:
+        """Returns the model properties as a dict
+
+        Returns
+        -------
+        Dict
+            Dictionary indexed by property name containing all the model properties
+        """
+        result = {}
+
+        for attr in self.swagger_types.keys():
+            value = getattr(self, attr)
+            if isinstance(value, list):
+                result[attr] = list(
+                    map(lambda x: x.to_dict() if hasattr(x, "to_dict") else x, value)
```

Comment:
If we're moving this can we un-python2 this? I suspect this can be a list comprehension here